### PR TITLE
Update UI of bnbridge to allow user to input MEMO, ERC20 ---> BEP2 

### PR DESCRIPTION
**Pull request by Renewwen**
Update UI of bnbridge to allow user to input MEMO, ERC20 ---> BEP2:
Before:
image
After:
image

### CODE REVIEW

**Comment by denniswon** [comment]
You need to cooridnate with Li on the UI. The current one is not good enough.

**Comment by lijiang2087** [comment]
@Renewwen come sync in person please :)

**Comment by Renewwen** [comment]
After coordinating with Li, I improved the UI to make it more easy to follow 
Prev: 
image
Updated:
image

File: bnbridge/src/App.js [hunk 1]
```diff
@@ -20,19 +19,18 @@ function App() {
       <CssBaseline />
       <Grid
         style={{
-          padding: '5% 30% 5% 10%',
```
Comment (denniswon) [inline bnbridge/src/App.js:23]: i intentionally added this. revert this.

File: bnbridge/src/App.js [hunk 2]
```diff
@@ -20,19 +19,18 @@ function App() {
...
+          padding: '5% 10% 5% 10%',
           minWidth: 'fit-content',
-          marginLeft: '10%',
```
Comment (denniswon) [inline bnbridge/src/App.js:25]: i intentionally added this. revert this.

File: bnbridge/src/App.js [hunk 3]
```diff
@@ -20,19 +19,18 @@ function App() {
...
         container
         justify="space-evenly"
         alignItems="center"
         direction="row"
         >
-        <Grid item style={{ padding: '0% 3%', display: 'table', }} align='center' xs={6}>
+        <Grid item style={{ display: 'table', }} align='center' xs={6}>
           <Instructions />
         </Grid>
-        <Grid item style={{ padding: '0% 3%', display: 'table', }} align="center" xs={6}>
```
Comment (denniswon) [inline bnbridge/src/App.js:35]: this as well. revert.

File: bnbridge/src/components/controller/controller.jsx [hunk 4]
```diff
@@ -142,14 +150,22 @@ class Controller extends Component {
...
-          <Tab label="Swap" />
-          {/* <Tab label="List" />
-          <Tab label="Issue" /> */}
+          <Tab style={{fontSize:'20px'}} label="Swap" />
         </Tabs>
         {tabValue === 0 && <Swap onIssue={ this.onIssue } showError={ this.showError } onCreateAccount={ this.onCreateAccount } />}
         {tabValue === 1 && <List onIssue={ this.onIssue } showError={ this.showError } />}
         {tabValue === 2 && <Issue onBack={ this.onIssueBack }  issueFee={ issueFee } showError={ this.showError } />}
+        
+        <div style={{  margin: '10px', marginTop: '20px' }} />
```
Comment (denniswon) [inline bnbridge/src/components/controller/controller.jsx]: indentation. Need to un-indent 1 tab.

File: bnbridge/src/components/controller/controller.jsx [hunk 5]
```diff
@@ -142,14 +150,22 @@ class Controller extends Component {
...
+
+      <Typography className={classes.action2}>
+        Please contact <a href="mailto:hello@harmony.one" target="_blank" rel="noopener noreferrer">Harmony team</a> for any additional support
+      </Typography>
+      <Typography className={classes.action2}>
+        Note: UI of this page is not optimized for mobile experience.
+      </Typography>
```
Comment (denniswon) [inline bnbridge/src/components/controller/controller.jsx]: remove these 3 lines please

File: bnbridge/src/components/instructions/instructions.jsx [hunk 6]
```diff
@@ -33,6 +33,18 @@ const styles = theme => ({
     display: 'inline-block',
     marginTop: "0.5rem"
   },
+  actionHighlight: {
+    fontSize: '1rem',
+    color: '#f44336',
+    display: 'inline-block',
+    marginTop: "0.5rem"
```
Comment (denniswon) [inline bnbridge/src/components/instructions/instructions.jsx]: single quote

File: bnbridge/src/components/instructions/instructions.jsx [hunk 7]
```diff
@@ -33,6 +33,18 @@ const styles = theme => ({
...
+  },
+  actionLink: {
+    fontSize: '1rem',
+    color: '#2196f3',
+    display: 'inline-block',
+    marginTop: "0.5rem"
```
Comment (denniswon) [inline bnbridge/src/components/instructions/instructions.jsx:46]: single quote

File: bnbridge/src/components/swap/swap.jsx [hunk 8]
```diff
@@ -531,6 +544,23 @@ class Swap extends Component {
...
+                  placeholder="eg: 107800300"
+                  
+                  value={ bnbMemo }
+                  error={ bnbMemoError }
+                  onChange={ this.onChange }
+                  disabled={ loading }
+                />
+
+                {/*
+                <a style={{fontSize:'12px', marginBottom: '16px'}} href='https://docs.binance.org/memo-validation.html#memo-validation'>Optional BEP12 Transfer MEMO</a>
```
Comment (denniswon) [inline bnbridge/src/components/swap/swap.jsx]: if not used, remove. no need to comment

File: bnbridge/src/components/swap/swap.jsx [hunk 9]
```diff
@@ -708,16 +739,21 @@ class Swap extends Component {
 
       return (
         <React.Fragment>
-          <Label label={'Swap direction'} overridestyle={{ marginTop: '12px', marginBottom: '12px' }} />
+          <Label label={'Swap direction'} overridestyle={{marginTop: '10px', marginBottom: '12px' }} />
```
Comment (denniswon) [inline bnbridge/src/components/swap/swap.jsx:735]: add 1 space in front of marginTop

**Review by denniswon — CHANGES_REQUESTED** [review]

**Comment by Renewwen** [comment]
Updated again
image

**Comment by denniswon** [comment]
awesome

**Pull request merged**